Demo/Extensions/DateFormatter+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+internal extension DateFormatter {
+    
+    // MARK: - Properties
+    
+    static let header: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "d MMMM yyyy"
+        
+        return dateFormatter
+    }()
+    
+    static let time: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH:mm"
```

Comment:
Shall we include seconds, even if just because lots of our test data arrives in a short space of time?

Reply:
Done